RELEASING.md

```diff
@@ -17,14 +17,28 @@ The password is your API key (not your password), which you can find via https:/
         ...
         <servers>
             <server>
-                <id>bintray</id>
-                <username>justinedelson</username>
+                <id>ossrh</id>
+                <username>kwin</username>
                 <password>*****</password>
             </server>
         </servers>
         ...
     </settings>
 
+These credentials are used to deploy to [OSSRH][OSSRH].
+
+In addition you need to setup [GPG](https://central.sonatype.org/pages/working-with-pgp-signatures.html) to create OpenPGP signatures. After installing https://www.gnupg.org/download/ you need to create key pair (if you don't have one yet) and make sure that the public key is distributed via hkp://pool.sks-keyservers.net.
+
+It is recommended that your private key is protected with a passphrase. You can persist the passphrase in the settings.xml as well
+
+     <server>
+         <!-- has the passphrase for the gpg signing in encrypted format: http://maven.apache.org/plugins/maven-gpg-plugin/sign-mojo.html#passphraseServerId -->
+         <id>gpg.passphrase</id>
```

Comment:
@kwin  is this right? Wouldn't this just be a `<properties>` in a ossrh `<profile>`?

Reply:
GPG and OSSRH deployment are not directly related. This is for storing your GPG passphrase, otherwise you have to explicitly parameterize the mvn release goals. You should not use a simple property as that cannot leverage the encryption (http://maven.apache.org/guides/mini/guide-encryption.html). As servers cannot be given below a profile I don't really get your remark...